app/services/schools/query.rb

```diff
@@ -74,7 +74,7 @@ def transient_in_partnership(contract_period_years)
     end
 
     def transient_mentors_at_school(contract_period_years)
-      return School.none if ignore?(filter: contract_period_years) || contract_period_years.blank?
```

Comment:
I think the blank here is because contract period year is required and not optional, maybe we need to make that clearer with a test?